jobs/bbs/templates/bbs.json.erb

```diff
@@ -123,15 +123,15 @@
   end
 
   if_p("diego.bbs.sql.db_connection_timeout") do |value|
```

Comment:
The spec has a default value of "30s" and "600s" for these. Maybe instead of always appending the unit, we detect if any units are provided and assume seconds if no units are present?